### ISSUE TITLE
New account

### ISSUE THREAD

**Issue by Graywolf6953** [hymns-for-worship/issues#72]
It will not let me create a new account with my email address it just sits there with the little circle rotating. Anyone else have this issue? It's seems nothing works for me getting very frustrating.

**Comment by rodydavis** [comment]
Did the latest update stop the loading issue?

**Comment by Graywolf6953** [comment]
Last update was Aug 18. App version is
2.3.1+719. Current plan is free. I subscribe but it will not confirm. I
created a completely new email and used it to create a new account. Did not
change anything. Since it will not confirm I cannot view sheet music. My
app was not working even before the last update. Been around a month now if
that helps. I gave up on restoring former purchases. I don't mind
subscribing if it would work.

**Comment by rodydavis** [comment]
I am working on testing a variety of methods as some seem to work but not completely:

https://stackoverflow.com/questions/75027717/confirm-subscription-plan-from-play-store-flutter

https://stackoverflow.com/questions/15918662/android-in-app-subscription-not-working

https://github.com/flutter/flutter/issues/107454

Also I have one in beta I am testing but for me it is working but when it goes live some people still have issues

**Comment by Graywolf6953** [comment]
I went to Google play and now it shows I have the subscription. I go to the
app...nope! It says I have subscribed but with each selection it says I
must purchase. Not sure what to do. I am on social security and can't play
with my money like this.
On one occasion, after 3 days Google refunded because it would not confirm.
I subscribe again and now it shows I have it with no pending of
confirmation.
